src/content/article/over-a-year-late-a-dollar-short-more-than-two-decades-of-kodi/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Over a Year Late, A Dollar Short - More Than Two Decades of Kodi!
+date: 2024-03-24T11:45:55.334Z
+author: Keith Herrington
+tags:
+  - Community
+featured_image:
+  src: /images/blog/xbmc-2003-1080p.webp
+  title: XBOX Media Player Home Screen, 2003
+  alt: Image of the original XBMP home screen and menu
+---
+Hi, everyone - I'm Keith, and while I now find myself President of the Kodi Foundation, my initial involvement with Kodi goes way back, as a humble user. Since our 20th anniversary came and went, I've been meaning to write a bit of a look back on how we got here - how *I* got here. This is the story of my journey.
+
+**In the Beginning...**
+
+In 2002, a couple guys saw the potential of being able to watch media on their own terms, through what was a new type of device: a "connected" TV device. They wanted to bring many things you'd traditionally need a computer for to the living room, operating it from the couch without needing a keyboard or mouse, controlling events through what they called a *10 foot interface* because, well, it was an interface you could sit 10 feet away from.
+
+While special-purpose devices already existed, they usually were things like DVD players, and they weren't ‘connected’ to anything else or even had the ability to hook up to a network. Indeed, even the TiVo, one of the most impressive devices of the time, lacked an Ethernet port. One of the creators of Samba, an open source interoperability application to share files between Linux and Windows, eventually figured out how to add a network port to a TiVo; however, even though it was the first mass marketed consumer Linux-based appliance, it was still very much a closed, walled garden.
+
+The Xbox changed all that. In 2001, Microsoft released their first console, leveraging their work on x86 platforms to build out a version of Windows specialized for running games. The familiarity allowed game developers to easily port to the new gaming console; the fact that Microsoft sold them at a loss, knowing they would make it back in game sales, ensured rapid consumer adoption. Luckily for the hackers out there, it was more similar to than different from something the majority were familiar with: the standard personal computer running Windows. Very quickly, folks like Bunnie figured out ways around the security, achieving the holy grail in record time for a console: the ability to run untrusted code.
```

Comment:
```suggestion
The Xbox changed all that. In 2001, Microsoft released their first console, leveraging their work on x86 platforms to build out a version of Windows specialized for running games. The familiarity allowed game developers to easily port to the new gaming console; the fact that Microsoft sold them at a loss, knowing they would make it back in game sales, ensured rapid consumer adoption. Luckily for the hackers out there, it was similar to something the majority were familiar with: the standard personal computer running Windows. Very quickly, folks like Bunnie figured out ways around the security, achieving the holy grail in record time for a console: the ability to run untrusted code.
```
The original was rather a brain-twister!

Reply:
Again, the voice of the author - it's a familiar idiom - but I take your point, it's not obvious to all. I've tried something in between the original text and your suggestion.

_Luckily for the hackers out there, there were more similarities than differences when compared to something they were already very familiar with: the standard Windows personal computer_